server/server.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
+const session = require("express-session");
+const app = express();
+const path = require("path");
+
+//Main route middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(require("morgan")("tiny"));
+
+app.use("/", express.static(path.join(__dirname, "../public")));
```

Comment:
This should go after the session stuff - because as is, this means that requests will come to your site that aren't assigned a session - you will serve static content before giving the requestor a session.